fulp_modules/features/jobs/deputy/deputy_gear.dm

```diff
@@ -29,7 +29,7 @@
 	name = "Deputy - Engineering"
 
 	head = /obj/item/clothing/head/fulpberet/engineering
-	ears = /obj/item/radio/headset/headset_dep
+	ears = /obj/item/radio/headset/headset_dep/engineering
```

Comment:
No this was intentional

Reply:
ok why

Reply:
because there's supposed to have one base that all others are subtypes of. I tried to keep them all consistent to one deputy but there were some things I didnt want subtypes to inherit from the base one, and used another (I think it was Supply/Engineering?)

Reply:
If you dont want specific features to be inherited the simplest way to do that is to just make a generic base without any of those features instead of experimenting to see which base has the least overlap. In many other cases that's just legitimately not possible, on top of being difficult to read. It's disorienting for there to a /beret/science and /beret/engineering but no /beret/supply, despite the fact that it exists.

This is one of those cases where it's best to be verbose for the sake of clarity

Reply:
revert this man 4 real dude i am gonna get very angry man